docs/draft/17-url-scheme.md

```diff
@@ -0,0 +1,41 @@
+---
+permalink: /spec/17
+parent: Draft specs
+title: 17/URL SCHEME
+---
+
+# 17/Defined Url Scheme
+
+> Version: 0.1.0
+>
+> Status: Draft
+>
+> Authors: Aleksandar Djenic <aleksandardjenic@status.im> Michal Iskierko <michal@status.im>
+>
+
+
+## Table of Contents
+
+ 1. [Url Scheme](#url-scheme)
+ 2. [Defined url](#defined-url)
+
+## Url scheme
+
+- internal: "status-app://"
+- external: "https://status.app"
+
+## Defined url
+
+| Name | Url | Description |
+| ----- | ---- | ---- |
+| User profile | `/u/[compressed_user_key or ens_name]` | Display user profile popup for user with `compressed_user_key` or `ens_name` |
```

Comment:
Specify public keys as URL fragments, placing them after `#`. In order to limit their exposure to hosting servers and wider network in general.

```suggestion
| User profile | `/u[?d=<data>]#<compressed_user_public_key or ens_name>` | Display user |
```